xptifw/unit_test/xpti_api_tests.cpp

```diff
@@ -3,21 +3,32 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-#include "xpti_trace_framework.hpp"
+#include "xpti/xpti_trace_framework.hpp"
 
 #include <gtest/gtest.h>
 #include <iostream>
 #include <set>
 #include <vector>
 
 static int func_callback_update = 0;
+static bool TPCB2Called = false;
```

Comment:
Why did you make this global?

Reply:
I need to somehow make sure that callback is called. The previous version of the test `const_cast`ed `user_data` argument, but that's a code smell, so I figured that global var will be lesser evil here.

Reply:
I see. Could you use the user_data argument to pass a value that is stored here? My concern is that we're not testing the user_data handling anymore.

Reply:
Oh, ok, I can make that change